tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -89,13 +89,13 @@ Feature: antivirus
   Scenario Outline: upload a file with the virus to a public share
     Given using <dav-path-version> DAV path
     And user "Alice" has created folder "/uploadFolder"
-    And user "Alice" has created a public link share with settings
-      | path        | /uploadFolder            |
-      | name        | sharedlink               |
-      | permissions | change                   |
-      | expireDate  | 2040-01-01T23:59:59+0100 |
-      | password    | %public%                 |
-    When user "Alice" uploads file "filesForUpload/filesWithVirus/<file-name>" to "/<new-file-name>" using the WebDAV API
+    And user "Alice" has created the following link share:
+      | resource           | uploadFolder             |
+      | space              | Personal                 |
+      | permissionsRole    | edit                     |
+      | password           | %public%                 |
+      | expirationDateTime | 2040-01-01T23:59:59.000Z |
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/<file-name>" to "/uploadFolder/<new-file-name>" using the WebDAV API
```

Comment:
I mean the scenario `Scenario Outline: upload a file with the virus to a public share` suggests that the When action must be done by a public user.

Reply:
or that could be also something to be covered? public can also add files with virus.